OpenRA.Mods.Common/UtilityCommands/LegacyMapImporter.cs

```diff
@@ -130,7 +129,7 @@ enum IniMapFormat { RedAlert = 3 } // otherwise, cnc (2 variants exist, we don't
 
 		public void ConvertIniMap(string iniFile)
 		{
-			using (var stream = GlobalFileSystem.Open(iniFile))
+			using (var stream = Game.ModData.ModFiles.Open(iniFile))
```

Comment:
*shrug*
In theory it should work with `File.Open()`, but I won't change it until someone who can test it confirms no side-effects.